### DISCUSSION TITLE
Storage Usage Compare with Elasticsearch

### DISCUSSION THREAD

**Discussion by jiejiecn** [openobserve/openobserve#1473] [Announcements]
### Describe the problem that you experienced

According to the docs related, Openobserve can save quite a lot storage size compare to  ES.

But under my test, the same type of logs stored in Openobserve as attached
About 1 doc = 14.5 Kbytes. That's over 10x times of the ingested! How could this happen?
![image](https://github.com/openobserve/openobserve/assets/11746870/683ce0ef-9b6a-47f6-b61a-49568a96fdd5)

![image](https://github.com/openobserve/openobserve/assets/11746870/d3f07a47-840c-4e68-9509-0bb49474cb8b)

My instance is running with local storage, not object storage.

### Enter the URL of the topic with the problem

https://openobserve.ai/docs/#how-does-openobserve-compare-to-elasticsearch

**Comment by prabhatsharma** [comment]
It's probably because you must have been ingesting data very slowly. How much data are you ingesting every hour? OpenObserve joins all the data in a particular hour and compresses it heavily, but if you ingest very little data each hour then you are not giving chance to OpenObserve to combine and compress data and the overhead of metadata is higher than the original data.

**Reply by prabhatsharma** [reply]
Also have you enabled any partitions? after partitioning, data should be large enough in each partition for OpenObserve to be able to compress data.

**Reply by jiejiecn** [reply]
Those logs were ingested in around 20hours, about 3500 logs/per hour. And I enabled the partition key on "_timestamp". I will do another test later.

**Comment by hengfeiyang** [comment]
@jiejiecn I saw that in your image you enabled `_timestamp` as a partition key, as a result, every micro second create a directory and create a file and in the file only one record. This setting actually it can't work.

Please delete the stream, and ingest data again without any partiton key settings.

The partition key should only enable the field that has a few distinct values, then the data that has the same value will be saved into the same file. if you don't understand what is this, please use it carefully.

**Reply by jiejiecn** [reply]
I'll try again later.
